src/views/payment-sheet-views/card-view.js

```diff
@@ -471,13 +471,22 @@ CardView.prototype._onBlurEvent = function (event) {
 
   classlist.remove(fieldGroup, 'braintree-form__field-group--is-focused');
 
-  if (isCardViewElement() && field.isEmpty) {
+  if (shouldApplyFieldEmptyError(field)) {
     this.showFieldError(event.emittedBy, this.strings['fieldEmptyFor' + capitalize(event.emittedBy)]);
   } else if (!field.isEmpty && !field.isValid) {
     this.showFieldError(event.emittedBy, this.strings['fieldInvalidFor' + capitalize(event.emittedBy)]);
   } else if (event.emittedBy === 'number' && !this._isCardTypeSupported(event.cards[0].type)) {
     this.showFieldError('number', this.strings.unsupportedCardTypeError);
   }
+
+  setTimeout(function () {
+    // when focusing on a field by clicking the label,
+    // we need to wait a bit for the iframe to be
+    // focused properly before applying validations
+    if (shouldApplyFieldEmptyError(field)) {
+      this.showFieldError(event.emittedBy, this.strings['fieldEmptyFor' + capitalize(event.emittedBy)]);
+    }
+  }.bind(this), 150);
```

Comment:
💯 Hopefully this doesn't need to change.

Reply:
Yeah, we did a bit of manual testing to find that comfortable number. Generally, this is a very corner case, and even if the timeout is somehow inefficient, it would not be causing any major issue. There is a very tiny visual penalty involved if we go over that timeout period for any reason.